### ISSUE TITLE
nadeshiko misreports missing aac

### ISSUE THREAD

**Issue by RX14** [deterenkelt/Nadeshiko#17]
```
$ ffmpeg -hide_banner -codecs | grep aac
 DEA.L. aac                  AAC (Advanced Audio Coding) (decoders: aac aac_fixed )
 D.A.L. aac_latm             AAC LATM (Advanced Audio Coding LATM syntax)
```

yet nadeshiko reports

```
$ nadeshiko ~/Downloads/VID_20190403_191751902.mp4
* System ffmpeg: n4.1.2
  libavutil   56.22.100
  libavcodec  58.35.100
  libavformat 58.20.100
* FFmpeg doesn’t support aac encoder.
* FFmpeg doesn’t support requested encoder(s).
--- Call stack -----------------------------------------------------------------
main, line 171 in /usr/bin/nadeshiko
check_util_support, line 304 in /usr/lib/nadeshiko/nadeshiko_stage02_arguments_processing.sh
err, line 586 in /usr/lib/nadeshiko/bahelite/bahelite_messages.sh
__msg, line 1 in /usr/lib/nadeshiko/bahelite/bahelite_messages.sh

```

**Comment by deterenkelt** [comment]
Looks like the `-codecs` output in that build of FFmpeg is slightly broken: it’s missing the lists of encoders. Updating to a more recent version should solve the issue.

```
libavutil      56. 26.100 / 56. 26.100
libavcodec     58. 47.102 / 58. 47.102
libavformat    58. 26.101 / 58. 26.101
```

```
$ ffmpeg -hide_banner -codecs | grep  aac                                                                  
 DEA.L. aac                  AAC (Advanced Audio Coding) (decoders: aac aac_fixed libfdk_aac ) (encoders: aac libfdk_aac )                                                                                                                 
 D.A.L. aac_latm             AAC LATM (Advanced Audio Coding LATM syntax)
```

This happens from time to time in the builds from git, I plan to replace checking the `-codec` output with `-h encoder=…`

**Comment by deterenkelt** [comment]
Solved in [v2.7](https://github.com/deterenkelt/Nadeshiko/releases/tag/v2.7).
